doc/whats_new.rst

```diff
@@ -45,6 +45,8 @@ Enhancements
 
 - Similarly, :func:`mne_bids.get_head_mri_trans` and :func:`mne_bids.update_anat_landmarks` gained a new ``kind`` parameter to specify which of multiple landmark sets to operate on, by `Alexandre Gramfort`_ and `Richard Höchenberger`_ (:gh:`955`, :gh:`957`)
 
+- Add support for ieeg data in the coordinate frame 'Pixels'; although MNE does not recognize this coordinate frame and so it will be set to "unknown" in the montage, MNE can still be used to analyze this kind of data, by `Alex Rockhill`_ (:gh:`976`)
```

Comment:
```suggestion
- Add support for iEEG data in the coordinate frame ``Pixels``; although MNE-Python does not recognize this coordinate frame and so it will be set to ``unknown`` in the montage, MNE-Python can still be used to analyze this kind of data, by `Alex Rockhill`_ (:gh:`976`)
```